<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettingsDialog</class>
 <widget class="QDialog" name="ProjectSettingsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GEMSFITS project setup</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project folder path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="projDir">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="projDirButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Project name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="projName">
     <property name="text">
      <string>myproj1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Database path</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="ejdbDir">
     <property name="text">
      <string>/EJDB</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QToolButton" name="ejdbDirButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Database file name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="ejdbName">
     <property name="text">
      <string>myprojdb1</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Experimental samples database collection name</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="experCollect">
     <property name="text">
      <string>experiments</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Fitting task cases database collection name</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLineEdit" name="taskCollection">
     <property name="text">
      <string>tasks</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>GEMS3K input file set path</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLineEdit" name="gemsDir">
     <property name="text">
      <string>/GEMS</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QToolButton" name="gemsDirButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
